data/migrations/20190815113751_groups_users.js

```diff
@@ -20,6 +20,8 @@ exports.up = function(knex) {
 			.onDelete("CASCADE")
 			.onUpdate("CASCADE");
 		groups_users.timestamps(true, true);
+
+		groups_users.unique(["user_id, group_id"]);
```

Comment:
These should be two separate strings like the one above, right?

Reply:
Fixed, sorry about that